external/storm-cassandra/src/test/java/org/apache/storm/cassandra/DynamicStatementBuilderTest.java

```diff
@@ -56,78 +61,93 @@
         when(mockTuple.getFields()).thenReturn(new Fields("weatherstation_id", "event_time", "temperature"));
     }
 
+    public static final String QUERY_STRING = "INSERT INTO weather.temperature(weatherstation_id,event_time,temperature) VALUES (?,?,?);";
+
```

Comment:
Change weatherstation_id to weather_station_id which is aligned with actual name in the table mentioned in resources.